kedro/framework/cli/project.py

```diff
@@ -234,3 +235,34 @@ def run(  # noqa: PLR0913
             pipeline_name=pipeline,
             namespace=namespace,
         )
+
+
+def _find_run_command(package_name: str) -> Any:  # pragma: no cover
```

Comment:
Should this still be an internal method if we expect this to be user-facing function?

Reply:
Good question. Initially I called `_find_run_command()` from `run()` (https://github.com/kedro-org/kedro/pull/4012/commits/651bd2f694bc5fda3486c028acca725f18ccf2ed) which was cleaner, but now having moved the find run logic to `cli/project.py` it would clash with the existing click run command. So I guess now it makes sense to make `_find_run_command()` public.

Reply:
I think it makes sense to make it public.

Reply:
Done!